source/_integrations/iotmeter.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: IoTMeter
+description: Instructions on how to integrate IoTMeter within Home Assistant.
+ha_category:
+  - Energy
+ha_release: 2024.6
+ha_iot_class: Cloud Polling
+ha_domain: iotmeter
+ha_platforms:
+  - sensor
+ha_config_flow: true
+ha_codeowners:
+  - '@lipic'
+ha_integration_type: integration
+---
+
+The **IoTMeter** {% term integration %} allows you to monitor and manage your IoT meters within Home Assistant. This integration provides real-time data and control over your connected devices.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The {% term integration %} adds the following sensors:
+
+- Total Power (unit: kW)
+  This sensor measures the total power being consumed at any given time. It is useful for monitoring overall power usage and identifying peak consumption periods.
+
+- Total Consumption Energy (unit: kWh)
```

Comment:
Is there a reason you changed from sentence-style to heading style capitalization? Is this how the UI string are written? We usually use sentence-style capitalization, also in headings.

Reply:
I changed it unknowingly. Should I correct it to sentence-style capitalization and push it again?

Reply:
Yes, please :)

Reply:
Done :-)